easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-foss-2020b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on STAR-2.7.7a-GCC-10.2.0.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'opt': True}
```

Comment:
I don't think you need to specify any toolchainopts. Optimized builds are the default no?

Reply:
True. I removed that line. Thanks. 
I have also expanded on the sanity checks to include more files to be checked and updated the comment at the top of the file.